backend/test/integration/test_invite.py

```diff
@@ -50,6 +51,138 @@ def test_send_invite_email(self, with_db, with_client):
             subscriber = repo.subscriber.get_by_email(db, invite_email)
             assert subscriber is not None
 
+    def test_send_invite_email_subscriber_already_exists(self, with_db, with_client, make_pro_subscriber):
+        """Ensures send_invite_email fails if subscriber already exists"""
+
+        os.environ['APP_ADMIN_ALLOW_LIST'] = '@example.org'
+
+        the_other_guy = make_pro_subscriber()
+
+        response = with_client.post(
+            '/invite/send',
+            json={'email': the_other_guy.email},
+            headers=auth_headers,
+        )
+        assert response.status_code == 400, response.text
+        data = response.json()
+        assert data['detail']['id'] == 'CREATE_SUBSCRIBER_ALREADY_EXISTS'
+
+    def test_send_invite_email_subscriber_fails(self, with_db, with_client):
+        """Ensures send_invite_email fails if invite email is invalid"""
+
+        os.environ['APP_ADMIN_ALLOW_LIST'] = '@example.org'
+
+        invite_email = 'hi'
+
+        response = with_client.post(
+            '/invite/send',
+            json={'email': invite_email},
+            headers=auth_headers,
+        )
+        assert response.status_code == 422, response.text
+        data = response.json()
+        assert 'not a valid email address' in data['detail'][0]['msg']
+
+    def test_get_all_invites_requires_admin(self, with_db, with_client):
+        """Ensures getting all invites requires an admin user"""
+
+        os.environ['APP_ADMIN_ALLOW_LIST'] = '@notexample.org'
+
+        response = with_client.get(
+            '/invite',
+            headers=auth_headers,
+        )
+        assert response.status_code == 401, response.text
+        data = response.json()
+        assert data['detail']['id'] == 'INVALID_PERMISSION_LEVEL'
+
+    def test_get_all_invites(self, with_db, with_client, make_invite):
+        """Ensures we can get all invites"""
+        os.environ['APP_ADMIN_ALLOW_LIST'] = '@example.org'
+
+        invites = [make_invite(owner_id=TEST_USER_ID) for _ in range(2)]
+
+        response = with_client.get(
+            '/invite',
+            headers=auth_headers,
+        )
+        assert response.status_code == 200, response.text
+        invite_list = response.json()
+        assert len(invite_list) == 2
+        today = datetime.today().strftime('%Y-%m-%d')
+        for next_invite in invite_list:
+            assert next_invite['owner_id'] == TEST_USER_ID
+            assert today in next_invite['time_created']
```

Comment:
It might be safer to parse the time_created as a date and test it within the datetime object.